junit/src/test/scala/MiscTest.scala

```diff
@@ -77,7 +71,7 @@ class MiscTest {
         assert(ex.getSuppressed.size > 0)
         assert(ex.getSuppressed.forall(_.isInstanceOf[MultipleOf37Exception]))
         assert(ex.i == 37)
-        assert(ex.getSuppressed.map(_.asInstanceOf[MultipleOf37Exception].i).toList == List(74, 148, 259, 518))
+        assert(ex.getSuppressed.map(_.asInstanceOf[MultipleOf37Exception].i).toList == List(/*74,*/ 148, 259, 518))
```

Comment:
I think that this should depend on how the scheduler exactly divided the tasks.
It's possible that the task corresponding to `74` never got scheduled (e.g. because `tryCancel` was called on the `ForkJoinTask` in an attempt to piggy-back it on the same worker).

I'd say that this assertion should check that all `i` are a multiple of `37`, but not make any specific assumptions about the exact exceptions.